freqtrade/freqtradebot.py

```diff
@@ -1939,6 +1942,10 @@ def update_trade_state(
         trade = self._update_trade_after_fill(trade, order_obj)
         Trade.commit()
 
+        strategy_safe_wrapper(
```

Comment:
You will probably want to move this to `_update_trade_after_fill` - within the first if.